src/Promotions/ServiceProvider.php

```diff
@@ -83,6 +84,8 @@ private function bootPluginUpsells()
             Hooks::addAction('admin_notices', WelcomeBanner::class, 'render');
             Hooks::addAction('admin_enqueue_scripts', WelcomeBanner::class, 'loadScripts');
         }
+
+        (new CampaignsWelcomeBanner)->render();
```

Comment:
@JoshuaHungDinh I know your still working on this but let's wrap this in 
 [is_admin()](https://developer.wordpress.org/reference/functions/is_admin/)
 to make sure it's only being checked in admin screens.  I made a mental note to myself to start remembering to do this more in the future so i'm passing it along to you 😄